src/tex/appendix.tex

```diff
@@ -49,6 +49,100 @@ \subsection{Benchmark corpus}
     \script{plot_statistics.py}
 \end{figure}
 
+The corpus of the questions in \chembench, as shown in \Cref{tab:chembench_corpus_topic}, can be divided according to which chemical topic they belong to.
+
+\begin{table*}[!t]
+    \centering
+        \caption{Examples for each of the topics considered in the evaluation of the \chembench corpus.}
+    \begin{tabular}{p{2.7cm}cp{10cm}}
+        \toprule
+        \textbf{Question Type} & \textbf{\%} & \textbf{Example Questions}  \\ 
+        \midrule
+        Chemical Preference & 35.0 & Imagine an early virtual screening campaign setting (accounting for simple aspects such as oral availability and small molecular profile, but no other modalities such as covalency or bifunctionality). Which of the following two candidates would you prefer for further development? \\
```

Comment:
the pedantic version of me would now also use `showyourwork` to compute those percentages, but I'd leave it for now as it is as we have more pressing problems. But it would be a cool enhancement to really have all numbers automatically updated.